generators/client-2/templates/src/main/webapp/app/_app.state.ts

```diff
@@ -1,5 +1,8 @@
 import { NavbarComponent } from './layouts';
-import { AuthService, <%=jhiPrefixCapitalized%>LanguageService } from './shared';
+import { AuthService } from './shared';
+<%_ if (enableTranslation) { _%>
```

Comment:
It gets applied during generation as the files are passed through this filter. I wouldnt want current regex changed as the function would be used for both ng1 and ng2 so i would add these as additional or conditions to the current regex to strip ng2 resolve blocks. for app.state its ok to have condition.